remoteit/desktop
# [DESK-730][DESK-643] Style update and Show log folder

### Description


Routing update
Providing more visual difference and hierarchy
Display Log and config folders

### Test plan



1. Sign in
2. Go to settings page
3. Click show logs 
4. You should see folders open with logs in them and config file

### Ticket



<https://remoteit.atlassian.net/browse/DESK-730>
<https://remoteit.atlassian.net/browse/DESK-643>
